hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2513,6 +2513,8 @@ class Emit[C](
     def emit(ir: IR, env: E = env): EmitCode =
       this.emitWithRegion(ir, mb, region, env, None)
 
+    def dEmit(ir: IR, env: E = env): EmitCode = emit(ir, env)
```

Comment:
What's this? It's just an alias of `emit`.

Reply:
There's an `emit` method on `NDArrayEmitter`, so I needed a different name to use within `outputElement`.